source/content/Character/Classes/Cleric/Miracles/Level 1/Detect Chaos.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Detect Chaos
+tags:
+  - level_1
+  - cleric
+---
+**Cleric Level 1**
+**Duration**: Instantaneous
+**Range**: Line of Sight
+
+This spell allows the caster to know if anything within his field of vision (or on his person) is **Chaotic**. For the purposes of this spell, Chaotic is defined as, but not limited to:  
+
+- **Undead creatures**
```

Comment:
btw do we count undead as chaotic?

Reply:
I guess we count them as Order?

Reply:
Well, shit

Reply:
too much to rewrite thou. we can:
* leave it as is for the playtest
* try to check everywhere for the undead stuff and change it. idk however how much of the stuff would break. chaotic undeat may be integral for many settings, modules and the like